src/matchers/asymmetric/A.ts

```diff
@@ -0,0 +1,36 @@
+import { AsymmetricMatcher } from './Base'
+
+interface Newable {
+  new (...args: any[]): any
+}
+
+/**
+ * Matches a instance of a class. It's works with primitives as expected (uses typeof)
+ */
+export class AMatcher extends AsymmetricMatcher {
```

Comment:
https://youtu.be/o9pEzgHorH0?t=129

Reply:
In this case, it's useful to have a common class acting as tag class to differentiate normal values from matchers for example in `toEquals`.